content/influxdb/cloud-dedicated/reference/release-notes/influxctl.md

```diff
@@ -11,6 +11,30 @@ menu:
 weight: 202
 ---
 
+## v2.6.0 {date="2024-03-18"}
+
+`influxctl` 2.6.0 introduces the ability to create, list, and revoke
+management tokens. These tokens allow users to authenticate directly without the
+need to for OAuth2.
```

Comment:
```suggestion
`influxctl` 2.6.0 introduces the ability to create, list, and revoke
management tokens and allow you to authenticate directly with your
InfluxDB cluster instead of an OAuth2 provider.
```